guides/common/modules/proc_importing-all-available-ansible-playbooks.adoc

```diff
@@ -0,0 +1,17 @@
+
+[id="importing-all-available-ansible-playbooks_{context}"]
+= Importing All Available Ansible Playbooks
+
+You can import all the available Ansible playbooks to {Project} from collections installed on {SmartProxy}.
+
+.Prerequisite
+* Ansible plug-in in {Project} is enabled
+
+.Procedure
+. Import the Ansible playbooks using the following API request:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+curl -X PUT 'Content-Type: application/json' https://_{foreman-example-com}_/ansible/api/v2/ansible_playbooks/sync?proxy_id=_My_{smart-proxy-context}_ID_ | json_pp
```

Comment:
Where does `json_pp` come from? We don't have that anywhere in our current docs.

Reply:
> Where does `json_pp` come from? We don't have that anywhere in our current docs.

from the demo https://www.youtube.com/watch?v=ykQFM7aFczY&t=1788s

Reply:
I think it's a developer short hand for JSON Pretty Print. Most people won't have that (on a related note, you can get the same by piping it to [jq](https://stedolan.github.io/jq/)). I'd leave it out